src/main/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactory.java

```diff
@@ -0,0 +1,140 @@
+  public static final String MAIN_RESOURCES = "src/main/resources";
+  public static final String LOGGING_TEST_CONFIGURATION = "logback.xml";
+  public static final String ORG_POSTGRESQL = "org.postgresql";
+
+  private final DockerImages dockerImages;
+
+  public PostgresqlModuleFactory(DockerImages dockerImages) {
+    this.dockerImages = dockerImages;
+  }
+
+  public JHipsterModule buildModule(JHipsterModuleProperties properties) {
+    Assert.notNull("properties", properties);
+
+    String packagePath = properties.basePackage().path();
+    String applicationName = properties.projectBaseName().capitalized();
+    JHipsterSource source = from(SOURCE);
+    DockerImage postgresqlDockerImage = dockerImages.get(POSTGRESQL_DOCKER_IMAGE_NAME);
+    JHipsterDestination databasePath = toSrcMainJava().append(packagePath).append(DEST_SECONDARY);
+
+    //@formatter:off
+    return moduleBuilder(properties)
+      .context()
+      .packageName(properties.basePackage())
+        .put("applicationName", applicationName)
+        .put("srcDocker", SRC_MAIN_DOCKER)
+        .put("postgresqlDockerImageWithVersion", postgresqlDockerImage.fullName())
+        .and()
+      .documentation(documentationTitle("Postgresql"), from("server/springboot/database/postgresql/postgresql.md.mustache"))
+      .files()
+        .add(source.template("DatabaseConfiguration.java"), databasePath.append("DatabaseConfiguration.java"))
+        .add(source.template("FixedPostgreSQL10Dialect.java"), databasePath.append("FixedPostgreSQL10Dialect.java"))
+        .add(source.template("FixedPostgreSQL10DialectTest.java"), toSrcTestJava().append(packagePath).append(DEST_SECONDARY).append("FixedPostgreSQL10DialectTest.java"))
+        .add(source.template("postgresql.yml"), toSrcMainDocker().append("postgresql.yml"))
+        .and()
+      .javaDependencies()
+        .add(springDataJpa())
+        .add(psqlDriver())
+        .add(psqlHikari())
+        .add(psqlHibernateCore())
+        .add(testContainer())
+        .and()
+      .springMainProperties()
+        .set(springDatasourceUrl(), propertyValue("jdbc:postgresql://localhost:5432/" + properties.projectBaseName().name()))
+        .set(springDatasourceUsername(), propertyValue(properties.projectBaseName().name()))
+        .set(springDatasourcePassword(), propertyValue(""))
+        .set(springDatasourceDriverClassName(), propertyValue("org.postgresql.Driver"))
+        .set(propertyKey("spring.jpa.database-platform"), propertyValue(properties.basePackage().basePackage() + ".technical.infrastructure.secondary.postgresql.FixedPostgreSQL10Dialect"))
+        .and()
+      .springTestProperties()
+        .set(springDatasourceDriverClassName(), propertyValue("org.testcontainers.jdbc.ContainerDatabaseDriver"))
+        .set(springDatasourceUrl(), propertyValue("jdbc:tc:" + postgresqlDockerImage.fullName()+ ":///" + properties.projectBaseName().name() + "?TC_TMPFS=/testtmpfs:rw"))
```

Comment:
Sorry for the mistake:
```suggestion
        .set(springDatasourceUrl(), propertyValue("jdbc:tc:postgresql" + postgresqlDockerImage.version()+ ":///" + properties.projectBaseName().name() + "?TC_TMPFS=/testtmpfs:rw"))
```